lib/templates/simple_template.dart

```diff
@@ -0,0 +1,28 @@
+import 'template.dart';
+import 'package:process_run/process_run.dart';
+
+class SimpleTemplate extends Template {
+  @override
+  Future<void> apply(String projectDir, Variables variables) async {
+    await run(
```

Comment:
Might be better to make use of the `File` API here and copy data between them using File pointers. Would be more cross platform.

Reply:
cool, I was unaware dart had these low level file handling stuff, I will update the suggestions